projects/ngx-drag-to-select/src/lib/utils.ts

```diff
@@ -58,7 +63,7 @@ export const getMousePosition = (event: MouseEvent) => {
 };
 
 export const getScroll = () => {
-  if (!document) {
+  if (!document || !document.documentElement) {
```

Comment:
Why is this needed, is it for Cypress?

Reply:
Typescript was complaining that I `documentElement` could be null, so I added the null check. Do you have another idea?

Reply:
No, I think we can leave it here.
As I said before, notice we won't fallback to `document.body` anymore and thus can be removed?


```ts
x: document.documentElement.scrollLeft || document.body.scrollLeft,
y: document.documentElement.scrollTop || document.body.scrollTop
```

Reply:
Hm, I checked it again... all of the sudden TS no longer complains. This is strange.